### PR TITLE
feat(doc): add "ng doc" command

### PR DESCRIPTION
**Pull request by shairez**
see #324

### CODE REVIEW

**Comment by Brocco** [comment]
Interesting, I was thinking of having documentation show right in the console, hadn't considered launching a browser.

That being said, I like the simplicity of this.

**Comment by hansl** [comment]
@Brocco; the end goal is to also have a man page for documentation. As of right now, documentation for ng2 is still being built so launching a browser is the second best we can do :)

**Comment by filipesilva** [comment]
The only thing to point out is that maybe these new files could be in TS, otherwise both the design and implementation lgtm.

File: addon/ng2/commands/doc.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* jshint node: true */
```
Comment (hansl) [inline addon/ng2/commands/doc.js]: Could you make this a `.ts` file?  We support TypeScript in angular-cli. After that you can remove the `use strict` below as all ts files are strict.

**Comment by Brocco** [comment]
@hansl of course this makes an excellent solution until we can get in-console docs

@filipesilva I agree w/ the idea of making these `.ts` files for consistency

File: addon/ng2/tasks/doc.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* jshint node: true */
```
Comment (hansl) [inline addon/ng2/tasks/doc.js]: Same as the command above, please make this a `.ts` file.

**Comment by hansl** [comment]
2 nits, but LGTM overall. Good work.

**Comment by shairez** [comment]
Thanks!
Changed it to .ts and removed the `use strict` and the hint line.

**Comment by hansl** [comment]
Good work Shai!

Not sure why Appveyor has started failing.  :shrug: LGTM. Do you have permission to merge? If so, you want to `git push upstream $(git symbolic-ref HEAD):master` (assuming you have the angular-cli repo as upstream) in order to merge. We don't use the button.

If you don't have the proper permissions I can merge for you.

**Pull request closed without merging**